app/javascript/components/rooms/room/RoomJoin.jsx

```diff
@@ -152,6 +155,9 @@ export default function RoomJoin() {
           )}
         </Card.Footer>
       </Card>
+      <div className="text-center text-muted mt-3"> { t('authentication.already_have_account') }
+        <Link to="/signin" className="text-link"> { t('authentication.sign_in') } </Link>
```

Comment:
This doesn't work if external accounts is connected - we should think of a better way to handle that case